test/fit_tests/common/fit_common.py

```diff
@@ -363,7 +363,7 @@ def restful(url_command, rest_action='get', rest_payload=[], rest_timeout=None,
                                        verify=sslverify,
                                        )
         if rest_action == "binary-put":
```

Comment:
At a certain point, we have to verify that the API calls we are making are actually valid with the intended format of the API call.  Using the restful interfaces allows us to test that content and return codes of the calls are correctly implemented.  This catches unintended changes to the APIs.  Using the swagger generated python library bypasses one layer of that by using the generated code.  To me, we should have at least one suite of actual API tests that verify in the expected user format off the box, and then most other tests could use the swagger libraries.  So I think we need to keep this in place for a base API specific test suite.